src/registerIcons.ts

```diff
@@ -0,0 +1,10 @@
+import { library } from '@fortawesome/fontawesome-svg-core';
+import {
+  faCheckSquare,
+  faShoppingBag,
+  faXmark,
+} from '@fortawesome/free-solid-svg-icons';
+
+export const registerIcons = () => {
```

Comment:
this is causing a new lint warning because there's no return type on this function. If you don't have it already I highly recommend getting the eslint extension for your editor